addon_service/configured_addon/polymorphic_serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework_json_api import serializers
+
+from addon_service.configured_addon.citation.serializers import (
+    ConfiguredCitationAddonSerializer,
+)
+from addon_service.configured_addon.models import ConfiguredAddon
+from addon_service.configured_addon.storage.serializers import (
+    ConfiguredStorageAddonSerializer,
+)
+
+
+class ConfiguredAddonPolymorphicSerializer(serializers.PolymorphicModelSerializer):
+    polymorphic_serializers = [
+        ConfiguredCitationAddonSerializer,
+        ConfiguredStorageAddonSerializer,
+    ]
```

Comment:
@felliott Also here. Might be a good idea to just scan this PR for similar things, in case I miss something.